src/main.js

```diff
@@ -4,13 +4,14 @@ const chalk = require("chalk")
 const flags = require("./flags.json")
 const { name, version } = require('../package.json')
 const BLOCK = "█"
-const STRING_LEN = process.stdout.columns
-const TERMINAL_HEIGHT = process.stdout.rows
+let STRING_LEN = process.stdout.columns
 const MINI_FLAG_DISTANCE = 10 // spaces from the left
 const ARGS = process.argv.slice(2)
+const FLAG_TYPE = ARGS[0]
+const FILL_TERM = ARGS[1] === "true" || ARGS[1] === "True"
```

Comment:
this is fine for now, ill prolly end up makin it a flag once i can think ~~straight~~ gay since its just a boolean

Reply:
yeah, I figured this was probably the easiest way to do this bc I didn't want to deal witth parsing arguments, if you want me to change it I can tho